ros2_batch_job/__main__.py

```diff
@@ -553,6 +553,10 @@ def need_package_from_pipy(pkg_name):
             # TODO(jacobperron): Until upstream issue is resolved https://github.com/pyreadline/pyreadline/issues/65
             pip_packages += ['git+https://github.com/osrf/pyreadline']
 
+            # Setuptools > 61 somehow have broken Windows Debug.  Pin it to 59.6.0 here which
+            # matches Ubuntu Jammy, and wait until upstream setuptools settles down.
+            pip_packages += ["setuptools==59.6.0"]
```

Comment:
I think this should supplant the cookbook configuration but it may make sense to update the cookbook to account for this as well. Especially if the problems this resolves occur during image build https://github.com/ros-infrastructure/ros2-cookbooks/blob/latest/cookbooks/ros2_windows/recipes/pip_installs.rb#L48-L52